### DISCUSSION TITLE
위키의 commit 컨벤션 규칙 추가

### DISCUSSION THREAD

**Discussion by greyHairChooseLife** [wanted-pre-onboarding-side-project/habit-tracker#34] [Suggestion]
## 제안
[위키 commit 컨벤션](https://github.com/wanted-pre-onboarding-side-project/habit-tracker/wiki/commit-%EC%BB%A8%EB%B2%A4%EC%85%98#%EA%B7%9C%EC%B9%99)의 '규칙' 부분에 아래 항목을 추가합니다.

**로컬 브랜치의 merge 커밋을 남기지 않기(무조건 fast-forward merge만 하도록 브랜치 관리 잘 하기)**

## 근거
멤버들 각자가 로컬 브랜치를 마음대로 관리 하도록 하는 한 편, 그로 인해 git history를 읽는데 어려움을 주지 않도록 하기 위함입니다.

## 참고
[관련 레거시 이슈](https://github.com/wanted-pre-onboarding-side-project/habit-tracker/issues/26)

**Comment by jiyeon2** [comment]
좋습니다!! :+1: :+1:  
다른 분들은 잘 하고 계실수도 있겠지만
저같은 경우 해당 문제를 잘 인지하지 못했는데
규칙으로 명시해주시면 위키 확인할 때마다 상기할 수 있어서 좋겠습니다!!
